src/app/store/rubrics.reducer.spec.ts

```diff
@@ -0,0 +1,258 @@
+import { DisplayableErrorInterface } from '../object-interfaces/displayable-error.interface';
+import { RubricInterface } from '../object-interfaces/rubric.interface';
+import {
+    ConnectionLost,
+    ConnectionMade,
+    CreateRubric,
+    CreateRubricError,
+    CreateRubricSuccess,
+    DeleteRubric,
+    DeleteRubricError,
+    DeleteRubricSuccess,
+    GetRubric,
+    GetRubricError,
+    GetRubrics,
+    GetRubricsError,
+    GetRubricsSuccess,
+    GetRubricSuccess,
+    SetPageTitle,
+    UpdateCurrentRubric,
+} from './rubrics.actions';
+import { rubricsReducer } from './rubrics.reducer';
+import { getInitialState } from './rubrics.state';
+
+describe('RubricsReducer', () => {
+    const { rubrics: initialState } = getInitialState();
+    const rubric: RubricInterface = { name: 'test' };
```

Comment:
Hard coding strings like `'test'` is a simple way to do this, but not very robust. Ideally, we'd like to test strings like this with more random input like a user would provide.

A simple way to do this is to use [faker.js](https://github.com/Marak/faker.js), which is already installed in our npm packages. See the section in their Wiki about [generating random text](https://github.com/Marak/faker.js/wiki/Text) or the way [we use it in the rubric-action-button-group test](https://github.com/thegraidenetwork/rubric-creator/blob/master/src/app/components/rubric-action-button-group/rubric-action-button-group.component.spec.ts#L6) for some good examples.

While in this case, the string being used is trivial, it's a good habit to get into.